komga/src/main/kotlin/org/gotson/komga/domain/service/LibraryScanner.kt

```diff
@@ -18,95 +26,238 @@ class LibraryScanner(
   private val fileSystemScanner: FileSystemScanner,
   private val seriesRepository: SeriesRepository,
   private val bookRepository: BookRepository,
-  private val bookLifecycle: BookLifecycle,
   private val mediaRepository: MediaRepository,
-  private val seriesLifecycle: SeriesLifecycle
+  private val seriesLifecycle: SeriesLifecycle,
+  private val fileHasher: XXHasher,
+  private val taskReceiver: TaskReceiver,
```

Comment:
We shouldn't need to send tasks from the services directly. It's a lifecycle responsability.